src/main/scala/day08.scala

```diff
@@ -0,0 +1,72 @@
+import java.io
+object day08 {
+//  Write a "new control structure" called skip that takes a list of whatever you want as a parameter in one parameter list,
+//  and a function that operates on the list members in a second parameter list. The control structure should execute the function on
+//  every other element in the list.
+  def skipOG(l1: List[String], fn: String => String): List[String] = {
```

Comment:
Note the exercise requires you to have two parameter lists:

```suggestion
  def skipOG(l1: List[String])(fn: String => String): List[String] = {
```